pylint/test/extensions/test_check_raise_docs.py

```diff
@@ -109,16 +109,35 @@ def my_func(self):
                 args=('RuntimeError', ))):
             self.checker.visit_raise(raise_node)
 
+    def test_ignore_spurious_sphinx_raises(self):
```

Comment:
This test will never fail. The check for raise docs will try to establish what exceptions the method raises. If it can then it will check that those are documented. As this method raises no exceptions, it won't check for the existence of raise docs.